pygn-mode.el

```diff
@@ -1752,6 +1761,35 @@ When called non-interactively, select the game containing POS."
   (push-mark (pygn-mode-game-end-position) t t)
   (goto-char (pygn-mode-game-start-position)))
 
+(defun pygn-mode-describe-annotation-at-pos (pos &optional do-copy no-error)
+  "Describe the annotation symbol at point in the echo area.
+
+When called non-interactively, describe the annotation
+symbol corresponding to POS.
+
+With `prefix-arg' DO-COPY, copy the description to the kill ring,
+and to the system clipboard when running a GUI Emacs.
+
+When NO-ERROR is set, noninteractively, do not signal an error
+when POS is not on an annotation symbol."
+  (interactive "d\nP")
+  (let ((annotation-node (pygn-mode--true-containing-node 'annotation pos)))
+    (if annotation-node
+        (let* ((annotation-text (buffer-substring-no-properties
+                                 (pygn-mode--true-node-first-position annotation-node)
+                                 (pygn-mode--true-node-after-position annotation-node)))
+               (description (or (gethash annotation-text pygn-mode-annotation-names) "Unknown"))
+               (full-description (format "%s - %s" annotation-text description)))
+          (when do-copy
+            (kill-new full-description)
```

Comment:
I thought `kill-new` doesn't affect system clipboard (and the docs seem to indicate the same), but `emacs -q`-ing it, it seems that it does.